tox.ini

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 [tox]
-envlist = pep8, pylint, py27, jenkins
+envlist = py27, pep8, pylint
```

Comment:
I get a pep8 error on this branch after installation:

<img width="1079" alt="screen shot 2015-09-23 at 2 03 22 pm" src="https://cloud.githubusercontent.com/assets/231148/10058775/13bfb6e0-61fc-11e5-912d-d205d28a6653.png">

I manually rm'd the .eggs directory, then ran tox -re pep8 again.  No errors.

Whatever created .eggs should delete it after it's finished.